.dotnet/example_code/DynamoDB/TryDax/06-DeleteTable.cs

```diff
@@ -23,7 +23,6 @@
  * specific language governing permissions and limitations under the License.
```

Comment:
Please insert proper copyright notice.

Reply:
Revised as per discussion to match approved samples in TryDax.zip.